src/KM_Controls.pas

```diff
@@ -105,22 +104,19 @@   TKMControl = class
     fOnClickRight: TPointEvent;
     fOnDoubleClick: TNotifyEvent;
     fOnMouseWheel: TNotifyEventMW;
-    fOnFocus: TNotifyEventFocus;
-    fOnControlMouseDown: TNotifyEventShift;
-    fOnControlMouseUp: TNotifyEventShift;
+    fOnFocusChange: TBooleanEvent;       // Focus or Unfocus

```

Comment:
fOnFocus renamed to fOnFocusChanged (it can be triggered not on focus, but also on unfocus)
Type TNotifyEventFocus was removed, because its a duplicate of TBooleanEvent. TBooleanEvent used instead